### DISCUSSION TITLE
Where has Scribble gone?

### DISCUSSION THREAD

**Discussion by githubUser01946** [Mikubill/sd-webui-controlnet#423] [Q&A] [answered]
I am trying to change scribbles into something other than fodder for the bin and wondered where scribble tick box had gone?!

I can select the 2 models with scribble in their name but above them there was a tick box that said `scribble mode (reverse color)` is it the same as the present `Invert Input Color` or something else?

**Comment by ljleb** [comment, marked as answer]
Yes, the `Scribble Mode` checkbox has been renamed to `Invert Input Color`. Its behavior has not changed, just the name.
